spec/takeaway_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  #instantiates the takeaway, and initalizes it with a menu arguement
+  subject(:takeaway) { described_class.new(menu: menu, basket: basket) }
+  let(:menu) { double(:menu, print: menu_example) }
+  let(:menu_example) { "Rice £5" }
+
+  it "displays menu" do
+    expect(takeaway.print_menu).to eq("Rice £5")
+  end
+
+  # it "receives the method to print basket" do
+  #   expect(basket).to receive(:print).once 
+  #   subject.print_basket
+  # end
+
+  let(:basket) { double(:basket, print: basket_example) }
+  let(:basket_example) { "Rice 100, Soup 20, Chefs Special 10" }
+  it "shows the basket" do
+    expect(takeaway.print_basket).to eq ("Rice 100, Soup 20, Chefs Special 10")
+  end
+
+  let(:dish) { double :dish }
+  let(:quantity) { double :quantity }
+
+  it "responds to 2 arguements" do
```

Comment:
Not sure about this test as I believe it's testing state rather than behaviour. Would suggest changing if possible.